net/ServerSocket.hpp

```diff
@@ -103,14 +103,11 @@ class ServerSocket : public Socket
         if (events & POLLIN)
         {
             std::shared_ptr<Socket> clientSocket = accept();
-            if (!clientSocket)
+            if (clientSocket)
             {
-                const std::string msg = "Failed to accept. (errno: ";
-                throw std::runtime_error(msg + std::strerror(errno) + ')');
-            }
-
-            LOG_TRC("Accepted client #" << clientSocket->getFD());
-            _clientPoller.insertNewSocket(std::move(clientSocket));
+                LOG_TRC("Accepted client #" << clientSocket->getFD() << ", " << *clientSocket);
```

Comment:
LOG_TRCA(Socket, ...) - the socket code should just work and not spam our logs.

Reply:
`LOGA_TRC(Socket,  .. ` OK